src/ignitus-Shared/Components/InternshipList/index.ts

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import * as T from '../emotionStyles/shared'
+import * as C from '../emotionStyles/colors'
+
+export const InternshipTitle = styled(T.Title)`
+  color:orange
```

Comment:
create a new `const` with hex code of `orange`  in `colors.js`

Reply:
Okay